### ISSUE TITLE
一个select的small error

### ISSUE THREAD

**Issue by liuww111** [yahoo/pure#502]
我喜欢使用微软雅黑的字体，但是当我使用的时候发现select的字体高度发生了变化。
I like to use Microsoft YaHei font, but when I use the select font height changes
案例(Case):
&lt;select id="stacked-state" style="
    font-family: "Microsoft YaHei";
"&gt;
            &lt;option&gt;分组一&lt;/option&gt;
            &lt;option&gt;分组二&lt;/option&gt;
            &lt;option&gt;分组三&lt;/option&gt;
        &lt;/select&gt;

**Comment by liuww111** [comment]
link：http://purecss.io/forms/，我在网站上面做了在线测试（I did online testing on the website.）

**Comment by liuww111** [comment]
my blog 我的博客（截图了案例）：http://www.cnblogs.com/RainbowInTheSky/p/4567689.html

**Comment by zechau** [comment]
I think this may work:
```css
.pure-form select {
  /* height: 2.25em; */
 padding: .4375em .6em;
  border: 1px solid #ccc;
  background-color: #fff;
}
```

**Comment by jamesalley** [comment]
@zechau Thanks for chiming in to help. 

.pure-form select has height on it in an attempt to try to get select elements to look good when horizontally aligned with other elements. It's a difficult problem because of the variation in the way browsers render select elements. If I remember correctly, in the end we tried to split the difference in getting it to look as consistent as possible, but one negative effect of setting height is what you see here.

I don't know if it's a bug...it's more a reflection of how difficult it can be to style certain elements. Date pickers and scrollbars are other examples of this problem space.

**Comment by liuww111** [comment]
Thank two place

**Comment by zechau** [comment]
@jamesalley Yeah, it is sure that styling certain elements like select is difficult, but setting select to fixed height is not a good idea, because it really hide some part of the content within it  regardless of what font it is set, this deserves to be called a bug.In addition, setting select height dose not look consistent in latest firefox.  There should be some other alternative solutions.

I found default select height is equal to input height pluses 1px of padding top and 1px of padding bottom in chrome and firefox. So I subtracted 1px in each padding top and padding bottom of .pure-form select. Then I got this: padding: .4375em .6em; This works better than seting height.
